public/views/content/details/tabs/detailsEdit.html

```diff
@@ -42,40 +42,45 @@
         </div>
 
         <div class="panel-body">
-            <div ng-show="showSeoDetails" class="am-fade-and-slide-top">
-                <div class="row">
-                    <div class="col-md-12">
-                        <label for="contentSeoTitle" class="control-label">{{ "SEO_TITLE" | translate }}</label>
+            <div ng-show="$root.showSeoDetails" class="am-fade-and-slide-top">
+
+                <div class="form-group">
+                    <div class="row">
+                        <div class="col-md-12">
+                            <label for="contentSeoTitle" class="control-label">{{ "SEO_TITLE" | translate }}</label>
+                        </div>
                     </div>
-                </div>
 
-                <div class="row">
+                    <div class="row">
 
-                    <div class="col-md-12">
-                        <input ng-model="activeTranslation.seoTitle" class="form-control" id="contentSeoTitle" type="text"/>
+                        <div class="col-md-12">
+                            <input ng-model="activeTranslation.seoTitle" class="form-control" id="contentSeoTitle" type="text"/>
 
-                        <p><i>
-                            <small>{{ 'SEO_TITLE_DESCRIPTION' | translate }}</small>
-                        </i></p>
-                    </div>
+                            <p><i>
+                                <small>{{ 'SEO_TITLE_DESCRIPTION' | translate }}</small>
```

Comment:
We should use Twitter Bootstrap "Help text" and using the aria-describedby attribute:

http://getbootstrap.com/css/#forms